templates/workspace_services/guacamole/guacamole-server/docker/services/oauth/run

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/with-contenv sh
+set -x
+echo >&2 "starting oauth2-proxy"
+
+tar zxvpf /etc/oauth2/oauth2-proxy-v7.2.1.linux-amd64.tar.gz
+
+chmod +x ./oauth2-proxy-v7.2.1.linux-amd64/oauth2-proxy
+./oauth2-proxy-v7.2.1.linux-amd64/oauth2-proxy --version
+
+cookiesecret=$(dd if=/dev/urandom bs=32 count=1 2>/dev/null | base64 | tr -d -- '\n' | tr -- '+/' '-_'; echo)
+
+./oauth2-proxy-v7.2.1.linux-amd64/oauth2-proxy \
+--provider oidc \
+--skip-provider-button \
+--cookie-secret "${cookiesecret}" \
+--oidc-issuer-url "${OAUTH2_PROXY_OIDC_ISSUER_URL}" \
+--upstream http://0.0.0.0:8080  \
+--email-domain "${OAUTH2_PROXY_EMAIL_DOMAIN}"  \
```

Comment:
Is there a reason we are passing values that are already set as environment variables, reading https://oauth2-proxy.github.io/oauth2-proxy/docs/configuration/overview#environment-variables there should not be the need?

Reply:
you are right that its redundant. I like to have it explicit, this way you can search it in the code...
and also when you see the command itself you see these params are used, so its more straighforward...